### ISSUE TITLE
solr minor version increment

### ISSUE THREAD

**Issue by pameyer** [IQSS/dataverse#5442]
solr 7.3.0 to 7.3.1

**Comment by pameyer** [comment]
PR https://github.com/IQSS/dataverse/pull/5443 ; since it appears to not be auto-linking.

**Comment by kcondon** [comment]
Discussed with Pete. His test was to run the integration solr tests on it. Open question is whether this is enough of a validation.

**Comment by pdurbin** [comment]
@kcondon I just ran the phoenix tests against develop (commit 571ec66) and if you want I could upgrade Solr to 7.3.1 on the phoenix server and re-run the tests. This wouldn't test the SolrJ upgrade in the app itself, though.

Another thought is to spin up the branch with `./ ec2-create-instance.sh -b 5442-solr731` and then ssh'ing into the ec2 instance, manually upgrading to Solr 7.3.1 (usually this is done by dataverse-ansible, pinging @donsizemore ) and running `conf/docker-aio/run-test-suite.sh http://localhost:8080` to run the tests on the ec2 instance itself.

Finally, there's no entry in `doc/release-notes`. Do we want one? I think before the break we were gong to talk about if we the new "developers can express the need for an addition to release notes by creating a file in /doc/release-notes" idea in http://guides.dataverse.org/en/4.10.1/developers/making-releases.html#write-release-notes

**Comment by kcondon** [comment]
@pdurbin Thanks for offering to test -I think running the tests are part of code review, right? I'm OK with Pete's report about the test for this one but something to consider for others going forward.
I'd prefer to test in my local test env in general rather than some other place. I would like to be reasonably sure it will work by someone checking release notes for the suggested upgrade to look for any red flags or upgrade recommendations, run integration tests as you'd mentioned but maybe with a comment about scope of them, then I'd set up and run in my test env, do a manual acceptance test on it and run that config while testing the rest of the sprint to get "burn in" testing.
There remains a question as to whether an infrastructure change is optional or required. In this case it would be optional as it doesn't break anything and the current config recommendations are fine for 7.3.0. 
I don't want to police the release-notes process, should that be part of code review?That said, what would it say? This upgrade is optional, see installation guide on how to install v7.3.1. Once installed, modify startup script to use the new version and do a reindex?

**Comment by pdurbin** [comment]
Regardless of whether it's optional or required I went ahead and opened https://github.com/IQSS/dataverse-ansible/issues/34 because I'd like the Ansible code used by our "spin up a branch" scripts to deploy the same version of Solr. At standup I mentioned I'd do this.

Good point about updating the startup script for people who are upgrading (as opposed to a new install). I agree that steps of this nature should be mentioned in the release notes.

**Comment by pdurbin** [comment]
I hope no one minds but I just added a little doc fix in 9d6e7ec44 because the current URL in the dev guide to list Solr fields is wrong. I added "collection1" to the URL like this:

Old:

```
[glassfish@uiswhlpt3621005 ~]$ curl http://localhost:8983/solr/schema/fields
<html>
<head>
   <title>Status page</title>
</head>
<body style="font-family: sans-serif;">
<p style="font-size: 1.2em;font-weight: bold;margin: 1em 0px;">Internal Server Error</p>
<p>The server encountered an unexpected condition which prevented it from fulfilling the request</p>
<p>You can get technical details <a href="http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.5.1">here</a>.<br>
Please continue your visit at our <a href="/">home page</a>.
</p>
</body>
</html>
[glassfish@uiswhlpt3621005 ~]$ 
```

New:

```
[glassfish@uiswhlpt3621005 ~]$ curl http://localhost:8983/solr/collection1/schema/fields | head -10
{
  "responseHeader":{
    "status":0,
    "QTime":0},
  "fields":[{
      "name":"_root_",
      "type":"string",
      "docValues":false,
      "indexed":true,
      "stored":false},
[glassfish@uiswhlpt3621005 ~]$ 
```
